src/NodeServices/GraphicsDataInterfaces.cs

```diff
@@ -211,6 +211,49 @@ public interface IGraphicItem
         void Tessellate(IRenderPackage package, TessellationParameters parameters);
     }
 
+    /// <summary>
+    /// an interface that defines items which have a transform property which is a 4x4 matrix
+    /// </summary>
+    public interface ITransformable
```

Comment:
new interface - I have not added a protogeometry reference to this interface (using coordinate systems for example) but I have added those as convenience  methods to HelixRenderPackage's implementation of this interface.